src/platform/forms-system/src/js/review/ReviewPage.jsx

```diff
@@ -27,36 +31,40 @@ class ReviewPage extends React.Component {
     focusElement('h2');
   }
 
+  renderDowntime = (downtime, children) => {
+    if (downtime.status === externalServiceStatus.down) {
+      const Message = this.props.formConfig.downtime.message || DowntimeMessage;
+
+      return <Message downtime={downtime} />;
+    }
+
+    return children;
+  };
+
   render() {
-    const { formConfig, pageList, path } = this.props;
+    const { formConfig, pageList, path } = this.props.route;
```

Comment:
This means that we can get these values from the `route` prop, since the `mapStateToProps` middleman is gone.